ontology/yaml/resources/HVAC/entity_types/DOAS.yaml

```diff
@@ -219,3 +219,18 @@ DOAS_SFVSC_EFVSC_HTWHLSTC_CHWVM_HWVM_SFM_FDPSM_ACO2C_SDM2X_EDM2X:
   - SDM2X
   - EDM2X
 
+DOAS_SFSS2X_SFVSC2X_EFSS2X_EFVSC2X_DX4SC_EDBPC_SSPC_SARC_ECON_HTWHLSTC:
+  guid: "a3451f6d-f562-4715-85be-2d169b4dc3cb"
+  description: "DOAS with dual supply and exhaust fans, four stage compressor, building pressure control and energy recovery wheel"
+  implements:
+  - DOAS
+  - SFSS2X
+  - SFVSC2X
+  - EFSS2X
+  - EFVSC2X
+  - EDBPC
+  - DX4SC
+  - ECON
```

Comment:
Since this is a DOAS unit, there won't be a mixed air temperature since it is direct outside air. The BMS lists this as mixed air which is a bit confusing. After discussing with Claire please create a new field, leaving_heat_recovery_wheel_temperature_sensor, add this to HTWHLSTC as opt uses and remove econ.

Reply:
Edit: Please move this to the canonical section and add the canonical boolean